src/bin/build.rs

```diff
@@ -32,7 +32,7 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
     }
 
     if args.is_present("no-create") {
-        book.create_missing = false;
+        book.create_missing = Some(false);
```

Comment:
Do you reckon it's worth adding a deprecation warning here as well?

Reply:
Yes, I think that's a good idea.